runtime/appruntime/exported/config/config.go

```diff
@@ -51,7 +51,7 @@ type Runtime struct {
 	RedisServers     []*RedisServer          `json:"redis_servers,omitempty"`
 	RedisDatabases   []*RedisDatabase        `json:"redis_databases,omitempty"`
 	Metrics          *Metrics                `json:"metrics,omitempty"`
-	ServiceDiscovery map[string]Service      `json:"services,omitempty"`
+	ServiceDiscovery map[string]Service      `json:"service_discovery,omitempty"`
```

Comment:
I'll need `services` next week so I can tell an instance what services it is hosting.